crates/uv-pypi-types/Cargo.toml

```diff
@@ -19,7 +19,7 @@ workspace = true
 uv-distribution-filename = { workspace = true }
 uv-fs = { workspace = true, features = ["serde"] }
 uv-git = { workspace = true }
-uv-normalize = { workspace = true, features = ["schemars"] }
+uv-normalize = { workspace = true }
```

Comment:
Does this need to be enabled by a schemars feature?

Reply:
good catch